packages/fast-components-react-base/src/slider/slider.schema.ts

```diff
@@ -3,7 +3,8 @@ export default {
     title: "Slider",
     description: "A slider component's schema definition.",
     type: "object",
-    id: "slider",
+    id: "@microsoft/fast-components-react-base/slider",
+    formPluginId: "@microsoft/fast-components-react-base/slider",
```

Comment:
can you make these schema changes in another PR, mark it as a "fix", if you do this it will bump these packages to a new minor version and convolute their changelogs.